mailbox/api/src/main/java/org/apache/james/mailbox/model/MessageAttachment.java

```diff
@@ -75,9 +75,6 @@ public MessageAttachment build() {
             if (isInline == null) {
                 isInline = false;
```

Comment:
Please no variable reallocation!

isInline should be an Optional<Boolean> to ease computation.